business_requirement_deliverable/models/business.py

```diff
@@ -0,0 +1,272 @@
+    )
+    uom_id = fields.Many2one(
+        comodel_name='product.uom',
+        string='UoM',
+        required=True
+    )
+    qty = fields.Float(
+        string='Quantity',
+        default=1,
+    )
+    resource_type = fields.Selection(
+        selection=[('task', 'Task'), ('procurement', 'Procurement')],
+        string='Type',
+        required=True,
+        default='task'
+    )
+    user_id = fields.Many2one(
+        comodel_name='res.users',
+        string='Assign To',
+        ondelete='set null'
+    )
+    business_requirement_deliverable_id = fields.Many2one(
+        comodel_name='business.requirement.deliverable',
+        string='Business Requirement Deliverable',
+        ondelete='cascade'
+    )
+
+    @api.multi
+    @api.onchange('product_id')
+    def product_id_change(self):
+        description = ''
+        uom_id = False
+        product = self.product_id
+        if product:
+            description = product.name_get()[0][1]
+            uom_id = product.uom_id.id
+        if product.description_sale:
+            description += '\n' + product.description_sale
+        self.description = description
+        self.uom_id = uom_id
+
+    @api.onchange('resource_type')
+    def resource_type_change(self):
+        if self.resource_type == 'procurement':
+            self.user_id = False
+
+    @api.multi
+    @api.constrains('resource_type', 'uom_id')
+    def _check_description(self):
+        if self.resource_type == 'task'\
+                and self.uom_id.category_id.name != 'Working Time':
+            raise ValidationError(_(
+                "When resource type is task, the uom category should be time"))
+
+
+class BusinessRequirementDeliverable(models.Model):
+    _name = "business.requirement.deliverable"
+    _description = "Business Requirement Deliverable"
+
+    sequence = fields.Integer('Sequence')
+    description = fields.Text('Description', required=True)
+    product_id = fields.Many2one(
+        comodel_name='product.product',
+        string='Product',
+        domain=[('sale_ok', '=', True)],
+        required=False
+    )
+    uom_id = fields.Many2one(
+        comodel_name='product.uom',
+        string='UoM',
+        required=True
+    )
+    qty = fields.Float(
+        string='Quantity',
+        store=True,
+        default=1,
+    )
+    resource_ids = fields.One2many(
+        comodel_name='business.requirement.resource',
+        inverse_name='business_requirement_deliverable_id',
+        string='Business Requirement Resource',
+        copy=True,
+    )
+    business_requirement_id = fields.Many2one(
+        comodel_name='business.requirement',
+        string='Business Requirement',
+        ondelete='cascade'
+    )
+    unit_price = fields.Float(
+        string='Sales Price'
+    )
+    price_total = fields.Float(
+        compute='_get_price_total',
+        string='Total revenue',
+    )
+    tax_ids = fields.Many2many(
+        'account.tax',
+        string='Taxes')
```

Comment:
Actually this is used when we create the quotations from the BR.
option 1: we keep the taxes here and copy to SQ in the wizard (currently implemented)
option 2: no tax here and we take the standard tax+fiscal position from the product in the wizard.
I like better option 2: we could implement it later on.